llvm/lib/CodeGen/AsmPrinter/DwarfUnit.h

```diff
@@ -34,6 +34,12 @@ class MCSymbol;
 /// source file.
 class DwarfUnit : public DIEUnit {
 protected:
+  // Label to denote the end of Abbreviation contribution of this compile unit.
+  MCSymbol *AbbrevOffset = nullptr;
+
+  // Label to denote the start of Abbreviation Section.
+  MCSymbol *AbbrevOffsetBegin = nullptr;
```

Comment:
Is this really the section? Why isn't it called AbbrevSectionBegin ro something like that?

Reply:
Yes it is the beginning for the section, the label difference will always be` AbbrevOffset - AbbrevOffsetBegin`

I am bad with names, `AbbrevSectionBegin` is a better name for sure